jps-plugin/src/org/jetbrains/jps/incremental/scala/LogFilterService.scala

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.jps.incremental.scala
```

Comment:
This allow to plugin to filter out compilation logs (e.g. if macro generate a lot of unwanted info messages)